### PR TITLE
Update getToken method in tests and Update interceptor sample readme, crt location

### LINKED ISSUE #2369: Rename 'generateUserAccessToken' function used in integration tests to get prod token
### Describe your problem(s)
What better if we can rename this to `generateUserAccessTokenProduction`. May be by adding an overloaded method

_Originally posted by @praminda in https://github.com/wso2/product-microgateway/pull/2366#discussion_r740759019_

---

This will change this method in all the testcases too (because it is used in every test file). Still okay to update this?

_Originally posted by @SuKSW in https://github.com/wso2/product-microgateway/pull/2366#discussion_r740760813_

---

how about having a overload method called `generateUserAccessTokenProduction`

_Originally posted by @praminda in https://github.com/wso2/product-microgateway/pull/2366#discussion_r740767172_

### How will you implement it

---

### LINKED ISSUE #2385: Update sample interceptor readme
### Description:
$subject

---

### PR DESCRIPTION
**Pull request by suksw**
### Purpose
Fixes #2369 
Fixes #2385 

### Automation tests
 - Unit tests added: Yes/No
 - Integration tests added: Yes/No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

**Review by renuka-fernando — APPROVED** [review]

**Review by praminda — APPROVED** [review]

File: samples/interceptors/.gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 bookstore/
+certs/
```
Comment (renuka-fernando) [inline samples/interceptors/.gitignore:2]: Maybe we have to commit certs to the repo, to try out the scenario on the fly without generating certs. Will add with later changes, +1 for now

**Comment by choreo-cicd** [comment]
[failed] dev-deployment : 20211115.4

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment : 20211115.12

**Comment by choreo-cicd** [comment]
[failed] stage-deployment : 20211117.2

**Comment by choreo-cicd** [comment]
[succeeded] stage-deployment : 20211117.3

**Comment by choreo-cicd** [comment]
[succeeded] prod-deployment : 20211119.1

**Pull request merged**